account_financial_report/view/report_template.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="account_financial_report_assets_backend"
+              name="account_financial_report assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <script type="text/javascript"
+                    src="/account_financial_report/static/src/js/account_financial_report_backend.js"/>        
+            <script type="text/javascript"
+                    src="/account_financial_report/static/src/js/account_financial_report_widgets.js"/>
+        </xpath>
+    </template>
+
+    <template id="account_financial_report_assets_common" name="account_financial_report_assets_common" inherit_id="web.assets_common">
+        <xpath expr="." position="inside">
+            <link rel="stylesheet" type="text/scss" href="/account_financial_report/static/src/scss/report.scss" />
```

Comment:
Why do you need to change this if you have already changed https://github.com/OCA/account-financial-reporting/pull/559/files#diff-1d943c35c14f57134d9ad64496c5f9b5R6?

Reply:
@pedrobaeza I just removed the SCSS from `account_financial_report.internal_layout` because useless (https://github.com/OCA/account-financial-reporting/compare/7130857bce83b818ac91fe01209f473e271a3759..7c8c6ad5406fa9318e5a7be70b346a0c99a96494)

Removing it from `web.assets_common` would make the HTML report style fail. Removing from `web.report_assets_common` would make the PDF  style fail

Reply:
But why don't you simply keep it as in current v11? I don't want to have a background in the screen for "View" option, and that's what is happening with scss compiled file. This is a large discussion and finally I have removed variables in v11, so no need to put scss, but simply css.

Any way, my fear putting this is that this alters other styles in the backend.

Reply:
@pedrobaeza so you propose to move back to CSS removing the use of `$o-view-background-color` and `$o-main-text-color`, right?

Reply:
Yes, and keep the 2 assets links as they were.

Reply:
Thanks, done